projects/aca-content/assets/i18n/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "APP": {
       "COPYRIGHT": "© 2017 - 2020 Alfresco Software, Inc. All rights reserved.",
+      "TITLE": "Workspace",
```

Comment:
we have https://github.com/Alfresco/alfresco-content-app/blob/develop/app/src/app.config.json.tpl#L28 for that